transactions/create_forwarder.cdc

```diff
@@ -0,0 +1,23 @@
+import NonFungibleToken from 0xNFTADDRESS
+import ExampleNFT from 0xNFTCONTRACTADDRESS
+import NFTForwarding from 0xNFTFORWARDERADDRESS
+
```

Comment:
Can you add some comments to this transaction describing what contract it uses and how it works? Thanks!

Reply:
added, see below!